ts/src/config.ts

```diff
@@ -126,6 +127,9 @@ export interface Config extends GoogleAuthOptions {
   // Duration of time profiles collected when using startLocal().
   // For testing with startLocal() only.
   localTimeDurationMillis?: number;
+
+  // Array of paths to files containing source maps.
+  sourcemapPaths?: string[];
```

Comment:
Does the user have to provide this for source maps to work? On the debug agent, we auto-scan the directory to find source maps. Could we do something similar here?

Reply:
It would good to document the scenarios where users may want to provide this config option. It would be good to document the default behaviour.

Reply:
Done.